deploy_snapshot.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Change to `#!/bin/bash -eux` as is used for the other `travis/*` scripts:

* `-e` to exit immediately if any command fails
* `-u` to treat uses of unset `$VARIABLES` as errors
* `-x` to verbosely show what commands are being run in the Travis CI job log